tests/test_factor_analyzer.py

```diff
@@ -113,14 +120,19 @@ def test_analyze_impute_median(self):
                              'B': [4, 8, np.nan, 10, 16, 18],
                              'C': [6, 12, 15, 12, 26, 27]})
 
-        expected = data.copy()
-        expected.iloc[2, 1] = np.median([4, 8, 10, 16, 18])
-        expected_corr = expected.corr()
-        expected_corr = expected_corr.values
+        # expected = data.copy()
+        # expected.iloc[2, 1] = np.median([4, 8, 10, 16, 18])
+        # expected_corr = expected.corr()
+        expected_corr = pd.DataFrame(
+            [
+                [1.      , 0.987534, 0.946014],
+                [0.987534, 1.      , 0.983488],
+                [0.946014, 0.983488, 1.      ]
+        ]).values
 
         fa = FactorAnalyzer(rotation=None, impute='median', n_factors=1)
         fa.fit(data)
-        assert_array_almost_equal(fa.corr_, expected_corr)
+        assert_array_almost_equal(expected_corr, fa.corr_)
```

Comment:
Why flip this around?

Reply:
I guess it doesn't really matter. I prefer to put the expected first, but the tests are actually inconsistent.